src/metabase/models/pulse.clj

```diff
@@ -229,7 +229,7 @@
   "Hydrate Pulse or Alert with the Fields needed for sending it."
   [notification :- (mi/InstanceOf Pulse)]
   (-> notification
-      (hydrate :creator :cards :dashboard [:channels :recipients])
+      (hydrate :creator :cards [:channels :recipients])
```

Comment:
There is no hydration method for `:dashboard`, in Toucan 2 this throws an error so I removed the key. Toucan 1 just silently ignores undefined hydration keys 😢